libs/core/src/global/styles/tokens/semantic/semantic.json

```diff
@@ -238,171 +238,5 @@
-    },
-    "type": "typography"
-  },
-  "body": {
-    "value": {
-      "fontWeight": "{font-weight.normal}",
-      "fontSize": "{font-size.100}",
-      "lineHeight": "{line-height.150}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-med": {
-    "value": {
-      "fontSize": "{font-size.100}",
-      "lineHeight": "{line-height.150}",
-      "fontWeight": "{font-weight.medium}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-semi": {
-    "value": {
-      "fontSize": "{font-size.100}",
-      "lineHeight": "{line-height.150}",
-      "fontWeight": "{font-weight.semi-bold}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-bold": {
-    "value": {
-      "fontSize": "{font-size.100}",
-      "lineHeight": "{line-height.150}",
-      "fontWeight": "{font-weight.bold}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-sm": {
-    "value": {
-      "fontWeight": "{font-weight.normal}",
-      "fontSize": "{font-size.087}",
-      "lineHeight": "{line-height.150}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-sm-med": {
-    "value": {
-      "fontWeight": "{font-weight.medium}",
-      "fontSize": "{font-size.087}",
-      "lineHeight": "{line-height.150}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-sm-semi": {
-    "value": {
-      "fontWeight": "{font-weight.semi-bold}",
-      "fontSize": "{font-size.087}",
-      "lineHeight": "{line-height.150}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-sm-bold": {
-    "value": {
-      "fontWeight": "{font-weight.bold}",
-      "fontSize": "{font-size.087}",
-      "lineHeight": "{line-height.150}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-xs": {
-    "value": {
-      "fontWeight": "{font-weight.normal}",
-      "fontSize": "{font-size.075}",
-      "lineHeight": "{line-height.100}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-xs-med": {
-    "value": {
-      "fontWeight": "{font-weight.medium}",
-      "fontSize": "{font-size.075}",
-      "lineHeight": "{line-height.100}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-xs-semi": {
-    "value": {
-      "fontWeight": "{font-weight.semi-bold}",
-      "fontSize": "{font-size.075}",
-      "lineHeight": "{line-height.100}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
-  },
-  "body-xs-bold": {
-    "value": {
-      "fontWeight": "{font-weight.bold}",
-      "fontSize": "{font-size.075}",
-      "lineHeight": "{line-height.100}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
```

Comment:
Is there a reason why we removed these?

Reply:
These were removed in a previous PR because I am redoing semantic tokens.